rbac/server/api/users.py

```diff
@@ -181,6 +178,7 @@
     txn_key, _ = await utils.get_transactor_key(request)
     txn_list = await create_delete_role_owner_txns(txn_key, next_id, txn_list)
     txn_list = await create_delete_role_admin_txns(txn_key, next_id, txn_list)
+    txn_list = await create_delete_role_member_txns(txn_key, next_id, txn_list)
     txn_list = create_delete_user_txns(txn_key, next_id, txn_list)
 
     if txn_list:
```

Comment:
On line 194, you left the roles_query function delete_role_member_by_next_id() function. With your changes, we will not need that code anymore.